python/ThirteenTeV/Higgs/HH/ResonanceDecayFilter_example_HHTo2B2G_poweg_pythia8_cff.py

```diff
@@ -0,0 +1,45 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+from Configuration.Generator.Pythia8PowhegEmissionVetoSettings_cfi import *
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+    args = cms.vstring('/afs/cern.ch/work/a/acarvalh/public/GridPacks_HH/13TeV/GF_HH_NonRes_SM_NLO/ggHH_slc6_amd64_gcc630_CMSSW_9_3_0_my_ggHH.tgz'),
```

Comment:
could you switch to the generic location of the gridpack ? Now that the cards have been approved in https://github.com/cms-sw/genproductions/pull/1496 I guess the gridpack can be uploaded ?

Reply:
I will wait for the testing iof the events